src/components/PokemonsGaleryCards/PokemonsGaleryCards.js

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect } from 'react'
+import { connect } from 'react-redux'
+import galeryStyles from './GaleryComponent.module.css'
+import { CardDeck } from 'react-bootstrap'
+import PokemonCard from '../PokemonCard'
+import { fetchPokemons, selectedPokemon, unselectedPokemons, countPokemon } from '../../redux/actions/pokemonsActions'
+import { getId, pokemonDataApi } from '../../utils'
+import { RingLoader } from 'react-spinners'
+import PokemonsModal from '../PokemonsModal'
+
+const PokemonsGaleryCards = ({ fetchPokemons, scrollCounter, pokemonsList, selectedPokemon, countPokemon, spinerFetching }) => {
+
+  const pokemonDescriptionUrl = (url) => {
+    return `${pokemonDataApi}pokemon-species/${url.split('/')[6]}/`
+  }
+
+  useEffect(() => {
+    fetchPokemons(scrollCounter)
+  }, [fetchPokemons, scrollCounter])
+
+  window.onscroll = (() => {
+    if (window.innerHeight + document.documentElement.scrollTop === document.documentElement.offsetHeight) {
+      countPokemon(scrollCounter)
+    }
+  });
+
+  return (
+    <div xs='auto' sm='auto' md='auto' lg='auto' className={galeryStyles.cardDeckContainer}>
+      < CardDeck >
+          {pokemonsList.map((pokemon) => {
+            return (
+              <div key={pokemon.name + getId(22)} onClick={() => {
+                selectedPokemon(pokemon, pokemon.url, pokemonDescriptionUrl(pokemon.url))
```

Comment:
it is recommended to create a function to handle this call, to make it clear for future developer what was the intention with this click, and to make it easier to change, the function can have pokemon (the complete object) as a parameter